src/collections/_documentation/platforms/go/goroutines.md

```diff
@@ -9,7 +9,7 @@ The easiest way to handle this, is to create a new `Hub` for every goroutine you
 
 Once cloned, `Hub` is completly isolated and can be used safely inside concurrent call. However, instead of using globally exposed methods, they should be called directly on the `Hub`.
 
-Here are two examples: one that is non-deterministic, would leak information between threads, and could trigger a concurrent-write panic; and one that is totally safe, and should be used instead.
+Here are two examples: one that is non-deterministic, would leak information between threads, and one that is totally safe, and should be used instead.
```

Comment:
```suggestion
Here are two examples: 
- a recommended deterministic call on `Hub` that is safe
- a discouraged non-deterministic call on `Hub` that would leak information between threads
```

Reply:
I'm assuming one is deterministic and the other is non-deterministic. :) If that's the wrong assumption, try:
"Here are two examples:
 - a recommended call on `Hub` that is safe
 - a discouraged non-deterministic call on `Hub` that would leak information between threads"

Reply:
You are correct, thanks! :)